pkg/handler/common_repositories.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/content-services/content-sources-backend/pkg/api"
+	"github.com/content-services/content-sources-backend/pkg/dao"
+	ce "github.com/content-services/content-sources-backend/pkg/errors"
+	"github.com/labstack/echo/v4"
+)
+
+type CommonRepositoriesHandler struct {
+	RepositoryDao dao.RepositoryConfigDao
+}
+
+func RegisterCommonRepositoriesRoutes(
+	engine *echo.Group,
+	dao *dao.RepositoryConfigDao,
+) {
+	rph := CommonRepositoriesHandler{RepositoryDao: *dao}
+	engine.GET("/common_repositories/", rph.listCommonRepositories)
+}
+
+// ListCommonRepositories godoc
+// @Summary      List Common Repositories
+// @ID           listCommonRepositories
+// @Description  get common repositories
+// @Tags         common,repositories
+// @Accept       json
+// @Produce      json
+// @Success      200 {object} api.CommonRepositoriesCollectionResponse
+// @Router       /common_repositories/ [get]
+func (rh *CommonRepositoriesHandler) listCommonRepositories(c echo.Context) error {
+	jsonConfig, err := os.ReadFile("./pkg/config/common_repositories.json")
```

Comment:
I don't think this will work within the container without actually copying the .json to  /pkg/config/common_repositories.json

Usually you'd have to embed like this:  https://github.com/content-services/content-sources-backend/blob/be2bf7c4d0f4588b6f6e028bf6b78a89ccd78d1a/pkg/external_repos/store.go#L11-L14

and then read it from there:  https://github.com/content-services/content-sources-backend/blob/be2bf7c4d0f4588b6f6e028bf6b78a89ccd78d1a/pkg/external_repos/store.go#L42-L58